### ISSUE TITLE
Cart button not showin in cart module

### ISSUE THREAD

**Issue by rbuelund** [redCOMPONENT-COM/redSHOP#5563]
When I add a product to the cart, then the Go to cart button in the cart module is not shown before I refresh the page. The product IS shown in the module though.

**Comment by rbuelund** [comment]
Bump...

**Comment by NguyenBao10** [comment]
Yes, because you enabled "Enable Add to Cart lightbox (Ajax)", right? then you can go to cart page by clicking "view cart" button in the pop-up. But we will consider to show "go to cart" button in cart module, too. It will have more options to go to cart page then. Thank you for your suggestion
